lib/osf-components/addon/components/search-result-card/styles.scss

```diff
@@ -11,6 +11,7 @@
 
     h4 {
         font-weight: bold;
+        margin-top: 5px;
```

Comment:
The bug shows issues with the User search card, doing this doesn't seem to fix the issue. The issues lies with the `div:last-child` selector on line 22. It is perhaps a good idea not to overuse such selectors.

Reply:
This fix had been for every search result card. I like the suggestion for user card only. I am merging this as requested as well with 1971 when fixed.